scopes/workspace/eject/eject-cmd.ts

```diff
@@ -1,24 +1,29 @@
 import { Command, CommandOptions } from '@teambit/cli';
 import { OutsideWorkspaceError, Workspace } from '@teambit/workspace';
 import ejectTemplate from '@teambit/legacy/dist/cli/templates/eject-template';
+import { COMPONENT_PATTERN_HELP } from '@teambit/legacy/dist/constants';
 import { Logger } from '@teambit/logger';
 import { InstallMain } from '@teambit/install';
 import { ComponentsEjector } from './components-ejector';
 
 export class EjectCmd implements Command {
   name = 'eject <component-pattern>';
-  description = 'replace components maintained in the workspace with their corresponding packages';
-  helpUrl = 'docs/components/exporting-components#ejecting-components';
+  description = 'eject component from the workspace, and install it instead as a regular npm package.';
```

Comment:
you are defining a thing and uses the thing's name in the definition?
can we use something else besides "ejects"? this creates recursion :D

Reply:
well the term eject wasnt invented by bit... :) 
But can go with remove if that sounds better

Reply:
yeah, but this eject is a bit-term.
remove the local component and replace with a dependency?

Reply:
dependency is more of a bit/component-y term.
"remove the component from the workspace and install it as a package dependency"?